lib/routes/rsshub/transform/html.ts

```diff
@@ -54,66 +55,86 @@ Specify options (in the format of query string) in parameter \`routeParams\` par
+                        link = linkEle.attr(routeParams.get('itemLinkAttr'));
+                    } else {
+                        link = linkEle.is('a') ? linkEle.attr('href') : linkEle.find('a').attr('href');
+                    }
+                    // 补全绝对链接
+                    link = link.trim();
+                    if (link && !link.startsWith('http')) {
+                        link = `${new URL(url).origin}${link}`;
+                    }
+
+                    const descEle = routeParams.get('itemDesc') ? item.find(routeParams.get('itemDesc')) : item;
+                    const desc = routeParams.get('itemDescAttr') ? descEle.attr(routeParams.get('itemDescAttr')) : descEle.html();
+
+                    const pubDateEle = routeParams.get('itemPubDate') ? item.find(routeParams.get('itemPubDate')) : item;
+                    const pubDate = routeParams.get('itemPubDateAttr') ? pubDateEle.attr(routeParams.get('itemPubDateAttr')) : pubDateEle.html();
+
+                    return {
+                        title,
+                        link,
+                        description: desc,
+                        pubDate,
+                    };
+                } catch {
+                    return null;
                 }
+            })
+            .filter((i) => !!i);
+
+        const itemContentSelector = routeParams.get('itemContent');
+        if (itemContentSelector) {
+            items = await Promise.all(
+                items.map(async (item) => {
+                    const response = await got({
+                        method: 'get',
+                        url: item.link,
+                    });
+                    const $ = load(response.data);
+                    const content = $(itemContentSelector).html();
+                    if (!content) {
+                        return item;
+                    }
+
+                    item.description = sanitizeHtml(content, { allowedTags: [...sanitizeHtml.defaults.allowedTags, 'img'] });
 
-                const descEle = routeParams.get('itemDesc') ? item.find(routeParams.get('itemDesc')) : item;
-                const desc = routeParams.get('itemDescAttr') ? descEle.attr(routeParams.get('itemDescAttr')) : descEle.html();
-
-                const pubDateEle = routeParams.get('itemPubDate') ? item.find(routeParams.get('itemPubDate')) : item;
-                const pubDate = routeParams.get('itemPubDateAttr') ? pubDateEle.attr(routeParams.get('itemPubDateAttr')) : pubDateEle.html();
-
-                return {
-                    title,
-                    link,
-                    description: desc,
-                    pubDate,
-                };
-            } catch {
-                return null;
-            }
-        })
-        .filter(Boolean);
-
-    return {
-        title: rssTitle,
-        link: url,
-        description: `Proxy ${url}`,
-        item: items,
-    };
-}
+                    return item;
+                })
```

Comment:
Please use `cache.tryGet` to cache the returned `item`

Reply:
updated